settings/language-css.cson

```diff
@@ -1,4 +1,6 @@
 '.source.css':
+  'core':
+    'useTreeSitterParsers': false
```

Comment:
/cc @Ben3eeE - I'm a bit hesitant to release this new parser in a hotfix release, so I've turned it off by default for now, just for CSS. This way, we'll get proper syntax highlighting in `style` tags in HTML and in tagged template literals with the word `styled` in JavaScript, which will address some regression that have been reported. But we won't risk introducing any more breakage, since this is a patch release.